### ISSUE TITLE
ERROR mtp: fatal error LIBUSB_ERROR_TIMEOUT on D810

### ISSUE THREAD

**Issue by Squitor** [puhitaku/mtplvcap#48]
Hi, i have encountered a similar issue on my D810 as some Users before me. 

```
./mtplvcap -debug server
[0000]  INFO usb: found: 04b0:0436
[0000]  INFO main: started
[0000] DEBUG lv: manufacturer = NIKON, product = NIKON DSC D810, serialnumber = 0000002009398
[0000] DEBUG lv: model didn't match, falling back to the generic model 
[0003] ERROR mtp: fatal error LIBUSB_ERROR_TIMEOUT; closing connection.
[0006]  WARN lv: frameCaptor: failed to obtain an image: LIBUSB_ERROR_TIMEOUT
[0006]  WARN lv: workerLV: failed to get live view status: mtp: cannot run operation GetDevicePropValue, device is not open
[0006]  WARN lv: workerLV: failed to get live view status: mtp: cannot run operation GetDevicePropValue, device is not open
[0007]  WARN lv: workerLV: failed to get live view status: mtp: cannot run operation GetDevicePropValue, device is not open
[0007]  WARN lv: frameCaptor: failed to obtain an image: mtp: cannot run operation MTP_WMPPD_PlaylistObjectPref, device is not open

```
The shutter stays closed and the mirror up mode doesn't do anything either.

**Comment by puhitaku** [comment]
@Squitor Thanks for reporting an issue. Which version did you use?

**Comment by Squitor** [comment]
I used version 1.4.1 and also tried both fixes. All came with a similar result.

**Comment by puhitaku** [comment]
Resolved in #80
